packages/src/context/reducer.ts

```diff
@@ -11,9 +11,18 @@ export type OverlayReducerAction =
 export function overlayReducer(state: OverlayData, action: OverlayReducerAction): OverlayData {
   switch (action.type) {
     case 'ADD': {
+      const isExisted = state.overlayOrderList.includes(action.overlay.id);
+
+      if (isExisted && state.overlayData[action.overlay.id].isOpen === true) {
+        throw new Error('The same overlayId exists.');
```

Comment:
```suggestion
        throw new Error('You can't open the multiple overlays with the same overlayId. Please set a different id.');
```

I think it is better to make it clear to say the user that you can't open the multiple overlays with the same id.